src/main/java/org/folio/utils/UserPermissionsUtil.java

```diff
@@ -84,4 +120,16 @@ private static boolean isManagePermissionRequired(Set<String> newAcqUnits, Set<S
     return !CollectionUtils.isEqualCollection(newAcqUnits, acqUnitsFromStorage);
   }
 
+    /**
+   * The method checks whether the user has the desired permission to update the fiscal year.
+   *
+   * @throws HttpException if user does not have fiscal year update permission
+   * @param newFiscalYearId     fiscal year id coming from request
+   * @param fiscalYearIdFromStorage fiscal year id from storage
+   */
+
+  private static boolean isApporved(String newStatus) {
+    return !CollectionUtils.isEqualCollection(newStatus, "Approved");
```

Comment:
if you want to compare strings you should not use collections, you can use equals method or StringUtils